tests/test_transform.py

```diff
@@ -201,10 +204,25 @@ def test_rowcol():
         assert rowcol(aff, 101985.0+400.0, 2826915.0) == (0, 1)
 
 
-def test_xy_rowcol_inverse():
-    # TODO this is an ideal candiate for
-    # property-based testing with hypothesis
-    aff = Affine.identity()
-    rows_cols = ([0, 0, 10, 10],
-                 [0, 10, 0, 10])
-    assert rows_cols == rowcol(aff, *xy(aff, *rows_cols))
+@given(precision=st.integers(min_value=1, max_value=16),
+       row=st.integers(min_value=0, max_value=100),
+       col=st.integers(min_value=0, max_value=100),
+       pxsz=st.floats(min_value=1.0, max_value=1000.0),
+       xoff=st.floats(min_value=0.0, max_value=1000000.0),
+       yoff=st.floats(min_value=0.0, max_value=1000000.0))
+def test_xy_rowcol_inverse(precision, row, col, pxsz, xoff, yoff):
+    """Roundtrip of xy and rowcol for range of inputs"""
+    aff = Affine(pxsz, 0.0, xoff, 0.0, -pxsz, yoff)
+    assert (row, col) == rowcol(aff, *xy(aff, row, col))
```

Comment:
Given a combination of precisions (ignored), rows, cols, and affine transformation matrices.